caluma/caluma_core/jexl.py

```diff
@@ -68,6 +68,17 @@ def __init__(self, *args, **kwargs):
         self.add_transform("debug", self._debug_transform)
         self._expr_stack = []
 
+        self.add_transform(
+            "mapby", lambda arr, key: [obj.get(key, None) for obj in arr]
+        )
+        self.add_binary_operator(
+            "intersects", 20, lambda left, right: any(x in right for x in left)
+        )
+        self.add_transform("min", lambda arr: min(arr))
+        self.add_transform("max", lambda arr: max(arr))
+        self.add_transform("sum", lambda arr: sum(arr))
+        self.add_transform("round", lambda num, ndigits=None: round(num, ndigits))
```

Comment:
Error handling is missing. It's absolutely possible that `None` is coming in as `num` or in an `arr`. Did you guys test this, except from the unit tests?

Reply:
Shouldn't we just crash in such a case? Or should we really do a catch-all and return None?

Reply:
No we definitely shouldn't. I don't know what the requirements are but it needs to be handled without crashing.